tests/test_loaders.py

```diff
@@ -186,13 +174,9 @@ def test_download(mocker, httpserver):
                 url = dataset.remotes[key].url
                 try:
                     request = requests.head(url)
-                    assert request.ok, "Link {} for {} does not return OK".format(
-                        url, dataset_name
-                    )
+                    assert request.ok, "Link {} for {} does not return OK".format(url, dataset_name)
```

Comment:
we've got different line lengths set on black 😬

Reply:
I think this is cause you only run black on the source, not tests. I can add an extra commit to have black run on the tests directory via tox.ini to this PR if you want.